bin/i18n/i18n_script_utils.rb

```diff
@@ -52,13 +52,10 @@ def self.with_synchronous_stdout
   def self.to_crowdin_yaml(data)
     ast = Psych.parse_stream(Psych.dump(data))
 
-    # Make sure we treat the strings 'y' and 'n' as strings, and not bools
-    yaml_bool = /^(?:y|Y|n|N)$/
     ast.grep(Psych::Nodes::Scalar).each do |node|
-      if yaml_bool.match node.value
-        node.plain = false
-        node.quoted = true
-      end
+      node.plain = false
```

Comment:
What happens when we _do_ have boolean nodes, or otherwise just non-string data?

Reply:
We are only looping through "Scalar" values:
> The content of a scalar node is an opaque datum that can be presented as a series of zero or more Unicode characters.


https://yaml.org/spec/1.1/#scalar/information%20model
This change won't modify any data which doesn't make sense to surround with double quotes, such as Collection/List/Array/Set.

I tried setting a breakpoint in the Scalar node loop to see if Psych processed any Scalars as having a specific non-string Tag (`node.tag`), but it wasn't detecting any. So it doesn't seem to be parsing values as `!!bool`, `!!int`, etc.
https://yaml.org/type/index.html

Here is an example of what happens to bools, ints, and floats:
```
  dayne_bool: false
  dayne_int: 42
  dayne_float: 3.14
```
becomes
```
  "dayne_bool": "false"
  "dayne_int": "42"
  "dayne_float": "3.14"
```

Reply:
Fantastic, thanks!